### PR TITLE
Doc update for KEP-3352 Aggregated Discovery

### PR DESCRIPTION
**Pull request by Jefftree**
https://github.com/kubernetes/enhancements/issues/3352

### CODE REVIEW

**Comment by mickeyboxell** [comment]
Hi @Jefftree, when you're ready, populate your placeholder PR with feature documentation and change the state of the PR from **draft** to **ready for review**. The official Docs deadline for PRs to be ready for review is tomorrow, March 21st. All docs PRs must be reviewed and ready to merge by April 4th. To mark a pull request as ready for review, navigate to the merge box and click Ready for review.

Regarding the content of your documentation, do your best to describe your feature and how to use it. If you need help structuring your documentation, ask in the #sig-docs Slack channel. For more information, please take a look at [Documenting for a release](https://kubernetes.io/docs/contribute/new-content/new-features/#for-developers-or-other-sig-members) to familiarize yourself with the documentation requirements for the release.

File: content/en/docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -158,6 +158,30 @@ Refer to the table below for accepted request headers.
...
+{{< feature-state state="beta"  for_k8s_version="v1.27" >}}
+
+Kubernetes offers beta support for aggregated discovery, publishing
+all resources supported by a cluster through two endpoints (`/api` and
+`/apis`) compared to one for every group version. Requesting this
+endpoint drastically reduces the number of requests sent to fetch the
+discovery for the average Kubernetes cluster. This may be accessed by
+requesting the respective endpoints with an Accept header indicating
+the aggregated discovery resource: `Accept:
+application/json;v=v2beta1;g=apidiscovery.k8s.io;as=APIGroupDiscoveryList`.
```
Comment (tengqm) [inline content/en/docs/concepts/overview/kubernetes-api.md:181]:
```suggestion
the aggregated discovery resource:
`Accept: application/json;v=v2beta1;g=apidiscovery.k8s.io;as=APIGroupDiscoveryList`.
```

**Comment by apelisse** [comment]
Looks good, thanks

**Comment by mickeyboxell** [comment]
@Jefftree would you mind rebasing when you get the chance?

File: content/en/docs/concepts/overview/kubernetes-api.md [hunk 2]
```diff
@@ -158,6 +158,31 @@ Refer to the table below for accepted request headers.
...
+
+Kubernetes offers beta support for aggregated discovery, publishing
+all resources supported by a cluster through two endpoints (`/api` and
+`/apis`) compared to one for every group version. Requesting this
+endpoint drastically reduces the number of requests sent to fetch the
+discovery for the average Kubernetes cluster. This may be accessed by
+requesting the respective endpoints with an Accept header indicating
+the aggregated discovery resource:
+`Accept:
+application/json;v=v2beta1;g=apidiscovery.k8s.io;as=APIGroupDiscoveryList`.
```
Comment (tengqm) [inline content/en/docs/concepts/overview/kubernetes-api.md:182]:
```suggestion
`Accept: application/json;v=v2beta1;g=apidiscovery.k8s.io;as=APIGroupDiscoveryList`.
```

**Pull request merged**